qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -264,6 +264,67 @@ def test_get_to_console(self):
         log.info("o_hash:{}".format(o_hash))
         assert(s_hash == o_hash)
 
+    def test_snap(self):
```

Comment:
Would be nicer to not to put `test_snap` under `TestGetAndPut`. Create a new class `TestSnapshots` instead?

Reply:
done ... separate class created